pkg/policy/api/utils.go

```diff
@@ -98,3 +98,18 @@ func ParseL4Proto(proto string) (L4Proto, error) {
 	p := L4Proto(strings.ToUpper(proto))
 	return p, p.Validate()
 }
+
+// MonitorRepresentation returns string representation
+// sent as notification to monitor
+func (rules Rules) MonitorRepresentation() string {
+	result := "labels: %s, descriptions: %s"
```

Comment:
I wouldn't include the descriptions. The labels should be useful on their own. Plus, this separates the descriptions and labels, so it's hard to correlate them.